.github/workflows/ci.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  # Every Monday at 11:30am UTC
+  schedule:
+    - cron: '30 11 * * 1'
+
+jobs:
+  ci:
+    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
```

Comment:
The README for gha-ci says to include a `name` key here. We should either include one in these repos or remove that from the readme on gha-ci